documentation/src/main/asciidoc/titles/upgrading/stories.adoc

```diff
@@ -1,7 +1,6 @@
 //Reference user stories or topics.
 //Please review authoring guidelines in the Contributors Guide.
 
-include::{stories}/assembly_upgrading.adoc[leveloffset=+1]
```

Comment:
-1 from me. In addition to your `changes.asciidoc`, I think we should keep the upgrade/change log in the upgrading guide as googling "infinispan upgrades" points directly to this page.

Reply:
the problem is that the "changelog" is huge, both in terms of raw size as well as number of headings (which pollute the menu). This means that the upgrade procedures listed at the end are drowned under all that noise. What if we just have a link from the upgrading guide to the changes guide:

*For a detailed list of changes that have occurred between versions please read the "changelog"*

Reply:
That sounds like a good compromise to me :+1: